rfcs/text/0008-threat-intel.md

```diff
@@ -33,6 +33,8 @@ Stage 1: Describe at a high level how this change affects fields. Which fieldset
     _Type of indicator as reprsented by Cyber Observable in STIX 2.0_
   * threat.description
     _Describes the type of action conducted by the threat._
+  * threat.tlp
```

Comment:
ECS tries to avoid abbreviations when possible. However, exceptions are made when the abbreviation is widely-used and known: `ip`, `geo`, `os`.

I just wanted to make this note as an observation; I'm not familiar enough with traffic light protocol's prevalence to vote one way or the other. 😄

Reply:
@ebeahan - `TLP` is a widely used term in cybersecurity, but has no relevance or meaning outside the cyber domain. Because we are using it within the `threat` fieldset can we keep it? we can change to `traffic_light_protocol` to stay consistent with our principles if needed.

Reply:
I see no reason why 'tlp' isn't acceptable if 'ip' is. If you're in the cybersecurity industry and using tlp, then it is as recognizable as 'ip' is for 'internet protocol'.

In reality, abbreviations are used everywhere in ECS - look at file.gid or file.mtime or file.uid

I think "threat.traffic_light_protocol" falls on the wrong side of a laugh test here. Nobody wants to index that as a field name who is realistically going to be working with threat.* data. It's just 'tlp' to those people. Please think of the analysts somewhat with field names length. Nobody wants to type threat.traffic_light_protocol for a field that will ever only contain keywords: red / amber  / green / white

Reply:
I wouldn't say any ideas fall on one side of or the other of a "laugh test". Analysts come with all levels of experience and exposure, and that's okay.

I do see @ebeahan's point, but I think that if a practitioner hasn't been exposed to the TLP concept, they likely wouldn't understand the value (`white`, `green`, `amber`, or `red`) either. 

I think that `threat.tlp` is probably a cleaner approach and, in the vein of industry terms, is a standard in CTI.

Reply:
I probably could have phrased it better. Subject matter expert sanity check? :)

Reply:
Speaking for myself here, I very often fall on the "laugh" side of a "laugh test" 😄  Laughing is so much more fun! 😉 

It was fair of @ebeahan to raise this, as this is always a question we ponder on field names. Since TLP is a widely recognized acronym for this concept, I'm in favor of keeping `.tlp` as the field name.

Reply:
++ Thanks for the feedback @shimonmodi @iankoetter @webmat 😃 

Since there's agreement for `.tlp`. I'll mark this note as resolved.